api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/RelatedPersonTranslatorImpl.java

```diff
@@ -62,9 +65,26 @@ public RelatedPerson toFhirResource(@Nonnull Relationship relationship) {
 		
 		Person omrsRelatedPerson = relationship.getPersonA();
 		RelatedPerson relatedPerson = new RelatedPerson();
-		
-		relatedPerson.setId(relationship.getUuid());
 		relatedPerson.setBirthDate(omrsRelatedPerson.getBirthdate());
+		relatedPerson.setId(relationship.getUuid());
+		
+		if (omrsRelatedPerson.getBirthdateEstimated() != null) {
+			if (omrsRelatedPerson.getBirthdateEstimated()) {
+				DateType dateType = new DateType();
+				Calendar calendar = Calendar.getInstance();
```

Comment:
Instead of using `Calendar` can we change this to use the `java.time` package classes?

Reply:
changed the calendar to java.time package classes.